Makefile

```diff
@@ -7,6 +7,7 @@ init:
 	mkdir -p artifacts
 
 deps:
+	bundle install
 	r10k puppetfile install --moduledir modules --verbose
 
 tools:
```

Comment:
This can be removed since all 'tools' are now part of bundle install.

Reply:
Sorry, I meant, `make tools` target can be removed.
`bundle install` can stay as part of `make deps`.

Because `make deps` already has `bundle install`, we don't need `make tools` for anything.

Also, side consistency, `make tools` is no longer used in any other repos since it confused the users between using `make tools` and `make deps`.